pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -139,7 +143,7 @@ func (r *ReconcileDirectImageMigration) Reconcile(request reconcile.Request) (re
 	}
 
 	if !imageMigration.Status.HasBlockerCondition() {
-		_, err = r.migrate(imageMigration)
+		requeueAfter, err = r.migrate(imageMigration)
```

Comment:
No longer throwing away the requeueAfter, so now if a requeue duration is sleected in phase logic it actually gets used.